ui/src/components/SituationCard.vue

```diff
@@ -69,25 +77,29 @@ const handleSituationSelected = () => {
 <style lang="scss" scoped>
 @import '@/styles/variables.scss';
 .card {
-	width: 270px;
-	height: auto;
 	display: flex;
+	width: 100%;
 	flex-direction: row;
 	background-color: #ffffff;
 	cursor: pointer;
 	border: 1px solid $border-grey;
+	height: 100%;
 	&:hover {
 		border: 1px solid $dark-blue;
 	}
-	&.selected {
-		border: 1px solid $dark-blue;
-	}
 
 	&.rejected {
 		background-color: #f3f3f3;
 		opacity: 0.4;
 	}
 }
+.description {
+	text-overflow: ellipsis;
+	-webkit-line-clamp: 3;
+	display: -webkit-box;
+	-webkit-box-orient: vertical;
```

Comment:
Do we need `-webkit` for this class? Looks like `-webkit-box` is non-standard and shouldn't be used in prod. There should be a way to add the ellipsis for a long description without it. Might be easier if `.description` were on a div instead of a span.

```
  white-space: nowrap; 
  width: 100px; 
  overflow: hidden;
  text-overflow: ellipsis; 
```